packages/ui/src/components/Node/TableNodeComponent.tsx

```diff
@@ -122,6 +122,7 @@ const TableNodeComponent = ({ id, data }: {
   const [offset, setOffset] = useState(0)
   const loaderRef = useRef(null);
   const tableRef = useRef<HTMLTableElement>(null);
+  const [isExecuteRun, setIsExecuteRun] = useState(false);
```

Comment:
Can this be renamed to something a bit more readable? Does it makes sense to call it "finishedExecution" or "isRunning"?

Reply:
Double check this according to other comment. Maybe I missunderstood the meaning and recommended a bad name 😅